viewer/api/models.py

```diff
@@ -10,7 +10,7 @@ def visible(self):
 
 class DataSet(models.Model):
     name = models.CharField(max_length=255, default='')
-    slug = models.CharField(max_length=50, unique=True)
+    slug = models.CharField(max_length=100, unique=True)
```

Comment:
Seems we might as well go for 255 -- it's not like we're taxing this db :)

Reply:
I thought I saw in normandy these were 50 characters but I can't find where that lives now. So yeah, I could blow it all the way out.